hack/tools/install_or_tools.sh

```diff
@@ -6,6 +6,8 @@
 cd "${0%/*}"
 source ./common.sh
 
+BUILD_FILE=lib/build
+VERSION_FILE=lib/version
```

Comment:
```suggestion
BUILD_FILE=lib/fzn-or-tools-build
VERSION_FILE=lib/fzn-or-tools-version
```
Maybe have more explicit file names, just in case another tool will also use the `lib` directory one day.